charts/rancher-logging/0.1.2/charts/log-aggregator/charts/log-aggregator-windows/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: log-aggregator-windows
+version: 0.0.1
```

Comment:
These versions don't line up with the versions mentioned in the directory structure